app/src/main/java/de/danoeh/antennapod/activity/WidgetConfigActivity.java

```diff
@@ -63,6 +76,9 @@ public void onProgressChanged(SeekBar seekBar, int i, boolean b) {
                 opacityTextView.setText(seekBar.getProgress() + "%");
                 int color = getColorWithAlpha(PlayerWidget.DEFAULT_COLOR, opacitySeekBar.getProgress());
                 widgetPreview.setBackgroundColor(color);
+                if (widgetMorePreview != null) {
```

Comment:
Is there a case where it actually is null?

Reply:
nope, removed